LICENSE-Apache

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
All Kotlin runtime contributors have agreed to also license their code under BSD-3, which is the overarching license in this repo, so this file should be removed.